DataBiosphere/data-explorer
# Check for deploy.json and export bucket

This is initialization for Export to Saturn feature. Next PR will write JSON entities to a
GCS file.

If deploy.json or export bucket is missing, a warning is printed in API server logs. Data Explorer UI still loads, in case people want to play around with Data Explorer UI. Then if they click the Export button, an error message is shown.

Tested locally:
- deploy.json missing - When I click export button, I see error
- bucket missing - When I click export button, I see error
- deploy.json and bucket exist - When I click export button, I'm redirected to Saturn. This confirms local auth works.

Test on GCP
- deploy.json - N/A, deploy/deploy-api.sh won't work if deploy.json is missing
- bucket missing - When I click export button, I see error
- deploy.json and bucket exist. When I click on export button, I'm redirected to Saturn. This confirms GCP auth works. See http://melchang-test-project-3.appspot.com/

CC @malathi13 FYI
